packages/tools/fetch-tool/src/test/test.spec.ts

```diff
@@ -0,0 +1,25 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/* eslint-disable max-len */
+
+import * as fetchArgs from "../fluidFetchArgs";
+import { fluidFetchInit } from "../fluidFetchInit";
+import { fluidFetchMessages } from "../fluidFetchMessages";
+
+const paramURL = "https://www.office.com/launch/fluid/content?auth=2&siteUrl=https:%2F%2Fmicrosoft-my.sharepoint-df.com%2Fpersonal%2Fanthonm_microsoft_com&drive=b!_XnMGqtB_EyzUvxXY3p7BUP07bmr8R1EjfELBvCmYllNhl5eInKQSZqCPXPSbmqp&item=01TOL2ZGOU43SP4KNMDVHJQDEAIRIN7ERZ&file=https:%2F%2Fmicrosoft-my.sharepoint-df.com%2Fpersonal%2Fanthonm_microsoft_com%2FDocuments%2FFluid%2520Framework%2520Summer%2520Vacation.fluid%3Fweb%3D1";
```

Comment:
I don't think we can put this link like this. You can add a unit test if you want just to test the method you changed.

Reply:
i agree. But to test `loadAllSequencedMessages` and `saveOps` I still need documentService which needs an URL.